src/components/MultiHopTrade/components/MultiHopTradeConfirm/MultiHopTradeConfirm.tsx

```diff
@@ -143,8 +143,12 @@ export const MultiHopTradeConfirm = memo(() => {
           <TradeSuccess
             handleBack={handleBack}
             titleTranslation={
-              isArbitrumBridgeWithdraw ? 'bridge.arbitrum.success.tradeSuccess' : undefined
+              isArbitrumBridgeWithdraw
+                ? 'bridge.arbitrum.success.tradeSuccess'
+                : 'trade.temp.tradeSuccess'
             }
+            buttonTranslation={'trade.doAnotherTrade'}
+            summaryTranslation={'trade.summary'}
```

Comment:
ditto OCD just because why not

Reply:
Sorry ser, what mean?

Reply:
looks like the original ocd comment didn't send, but was about removing the JS bit and keeping these explicit strings i.e:
```suggestion
            buttonTranslation='trade.doAnotherTrade'
            summaryTranslation='trade.summary'
```

Resolving as turbo OCD